<?xml version="1.0"?>

<launch>
	<node pkg = "final_assignment" type = "second_option.py" name = "self_driving" output = "screen"/>
</launch>

